src/Controls/samples/Controls.Sample.Droid/MainApplication.cs

```diff
@@ -12,5 +12,15 @@ public class MainApplication : MauiApplication<Startup>
 		public MainApplication(IntPtr handle, JniHandleOwnership ownership) : base(handle, ownership)
 		{
 		}
+
+#if NET6_0_OR_GREATER
+		// TODO: https://github.com/dotnet/runtime/issues/51274
+		public override void OnCreate()
+		{
+			Java.Lang.JavaSystem.LoadLibrary("System.Security.Cryptography.Native.OpenSsl");
```

Comment:
FYI, this library was renamed to `libSystem.Security.Cryptography.Native.Android.so`: 

https://github.com/xamarin/xamarin-android/commit/bcb7637498bbdd7b8c1add56dcf1eebfc9038ada